Libplanet.Net.Tests/Messages/MessageValidatorTest.cs

```diff
@@ -70,6 +71,7 @@ public void ValidateAppProtocolVersion()
             var peer = new BoundPeer(trustedSigner.PublicKey, new DnsEndPoint("0.0.0.0", 0));
             var trustedPing1 = new PingMsg() { Remote = peer, Version = trustedApv1 };
             var trustedPing2 = new PingMsg() { Remote = peer, Version = trustedApv2 };
+            var trustedPing3 = new PingMsg() { Remote = peer, Version = trustedApv3 };
             var unknownPing1 = new PingMsg() { Remote = peer, Version = unknownApv1 };
             var unknownPing2 = new PingMsg() { Remote = peer, Version = unknownApv2 };
```

Comment:
It's really hard to keep track of what conditions are being tested. I would suggest one of the following:

- Use more descriptive variable names for different scenarios, as it isn't immediately apparent what is being tested around the `Assert` statements.
- Parameterize arguments.
- Add more comments below.
- Order tested scenarios below to match the order of variable declaration here.

Reply:
I agree, I had written test code that was hard to understand
thanks, I'll coding your suggestion

Reply:
I modified the tests... could you check this? 🙏